src/visual_editor/ElementDetails/DetailsRow.tsx

```diff
@@ -109,6 +112,13 @@ const DetailsRow = ({
           {_value}
         </div>
       )}
+      {onErase && (
+        <div className="px-1">
+          <button onClick={onErase}>
+            <RxCross2 className="w-4 h-4 cursor-pointer" />
```

Comment:
To me the X seems ambiguous. Does it erase everything inside, delete the element, undo / revert? Maybe instead `BsArrowCounterclockwise`

Reply:
When using the visual editor, we use the X in multiple places, and what it generally is used for is to remove the object (html attribute, class name) .. In this case, it's removing the *inner html changes* - it will effectively revert the inner html to its original value

I kind of want to keep the 'X' because visually it is consistent with other icons that do similar operations on the Visual Editor panel. However I could see the arrow making more semantic sense, though I think it might create some disruption visually. Do you feel strongly about using a different icon?

Reply:
I agree with Bryce that an undo arrow is more intuitive here.  My first thought when seeing the `x` was that it was going to delete the innerHTML (set it to an empty string).